### DISCUSSION TITLE
Issues with SeeedStudio XIAO SAMD21 modul

### DISCUSSION THREAD

**Discussion by i-g-g-y** [sandeepmistry/arduino-LoRa#615] [Q&A]
Hello, 

I have succesfull implementation on library for ESP32, ESP8266, atmega328, atmega32u4, but no success for XIAO SAMD21 module. 

I have working SPI and can dump LoRa registers OK, but during Tx, the modem does not return successful Tx status. Did anyone try doing this with XIAO board?

**Comment by Kongduino** [comment]
I did, with a Xiao BLE. I can try with the SAMD too...

![Xiao Ra-02](https://user-images.githubusercontent.com/35363882/204255052-a513a069-4812-4c18-8357-8165350a6897.png)

**Reply by i-g-g-y** [reply]
Thanks, must be a bug in my code then.

**Comment by i-g-g-y** [comment]
Please see my comments in "Issues". Turns out it was not my code.
